lib/datapacksjob.js

```diff
@@ -2961,12 +2963,69 @@ DataPacksJob.prototype.downloadPerformanceData = async function (jobInfo) {
                 records.push(record);
             })
             .on("end", async function () {
-                var res = records.map(item => {
-                    return item[field_name];
+               
+                var omni_id =records.map(item => {
+                    return JSON.parse(item[field_name]).OmniScriptId;
                 });
 
-                VlocityUtils.log(`${i} of ${totalEntries.totalSize}`);
+                var unique_omni_id=Array.from([...new Set(omni_id)]);
+ 
+                var unique_omni_id_format="";
+                
+                for(var j=0;j<unique_omni_id.length;j++){
+                    if(j == (unique_omni_id.length-1)){
+                        unique_omni_id_format=unique_omni_id_format+`'${unique_omni_id[j]}'`
+                    }else{
+                        unique_omni_id_format=unique_omni_id_format+ `'${unique_omni_id[j]}',`;
```

Comment:
Will need a max length for this string and to do a loop here or it will fail when over 20000 chars.

Reply:
As the limit for characters inside where clause in sosql is 4000, length of "unique_omni_id" is splitted into chunks of 150 to limit length of characters inside sosql where clause.